src/markitdown/_markitdown.py

```diff
@@ -676,19 +677,27 @@ def convert(self, local_path, **kwargs) -> Union[None, DocumentConverterResult]:
 
 class PdfConverter(DocumentConverter):
     """
-    Converts PDFs to Markdown. Most style information is ignored, so the results are essentially plain-text.
+    Converts PDFs to Markdown. Most style information is ignored, so the results are essentially plain-text.    
     """
 
     def convert(self, local_path, **kwargs) -> Union[None, DocumentConverterResult]:
+        """
```

Comment:
I think `pdf_engine` should be a named parameter in the function instead of being accessed via `kwargs`. This provides more clarity and ensures better handling of default values

Reply:
Thanks for you suggestion. I have added it to a named parameters ( I was meant to align with other converters' parameter definition), and added the exception case when `pdf_engine` is not valid. The new test cases could be seen on my new commit.

Reply:
This is optional, but adding one more named parameter could make this more customizable, like:

```py
def convert(self, local_path, engine: Literal['pdfminer', 'pymupdf4llm']='pdfminer', engine_kwargs=None, **kwargs) -> Union[None, DocumentConverterResult]:
```